application/src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -542,6 +551,15 @@ type Entrance {
   wheelchairAccessible: WheelchairBoarding
 }
 
+"A single use of an escalator."
+type EscalatorUse {
```

Comment:
Good idea to separate the movement on the escalator from the actual entity. We may or may not add it later on.